Resources/Prototypes/Entities/Mobs/NPCs/elemental.yml

```diff
@@ -0,0 +1,208 @@
+- type: entity
+  save: false
+  abstract: true
+  id: MobElementalBase
```

Comment:
you can get rid of like half of these if you use composition to inherit from the base mob prototypes. Look at how BaesSimpleMob does it.

Reply:
wouldnt i also inherit like blood solution etc?

Reply:
I checked all base types and like drones i think elementals need their own base so they dont inherit stuff like flammable and body solutions.